src/ActionList/Item.tsx

```diff
@@ -13,15 +15,39 @@ import {GroupContext} from './Group'
 import {ActionListProps, ListContext} from './List'
 import {Selection} from './Selection'
 import {ActionListItemProps, getVariantStyles, ItemContext, TEXT_ROW_HEIGHT} from './shared'
-import {LeadingVisual, TrailingVisual} from './Visuals'
+import {LeadingVisual, TrailingVisual, VisualProps} from './Visuals'
 
 const LiBox = styled.li<SxProp>(sx)
 
+const InactiveIndicator: React.FC<{
+  text: TooltipProps['text']
+  visualComponent: React.FC<React.PropsWithChildren<VisualProps>>
+}> = ({text, visualComponent: VisualComponent}) => (
+  <Tooltip text={text}>
```

Comment:
Just reading through [the accessibility section on conveying inactive state to screen reader users](https://primer.style/components/action-list#tooltips-and-dialogs-on-inactive-items), and wanted to comment to make sure we are rending the right type of Tooltip here. In the new Tooltip, there are [two types](https://primer.style/components/tooltip#options): label type and description type. In general, icon buttons are label type. (The type matters because we do the internal connection between the trigger and the tooltip using `aria-labelledby` or `aria-describedby` depending on the type) 

In this usage, the button doesn't have an accessible name and it is described by the tooltip text. Is that expected? Or do we want to use the tooltip text to label the visual component?

Reply:
Great question. I don't think I got specific guidance from a11y about the "type" of Tooltip to use. @alexislucio - what do you think?

Reply:
Hey hi, filling in for @alexislucio while they're away. I think we'd want `aria-describedby`. My thinking is with the current code:

1. The `button` that wraps the `svg` icon uses an `aria-describedby` to point to the tooltip's `id`.
2. The `svg` does not have an accessibile name, as it has `aria-hidden="true"` applied to it.
3. The tooltip's content would read something like, "Not available due to an outage"

This leaves the `button` without an accessible name. What we can do about this is point to the ActionList item's title (ex: "Octicons") and use `aria-labelledby` to provide the accessible name for the tooltip button (ex: `<button aria-labelledby="actionlist-item-title-octoicons">`). 

We can then use `aria-describedby` to supplement the announcement, so it could read something along the lines of:

```text
Octicons. Button. Not available due to an outage.
```

Reply:
Thanks @ericwbailey ! I updated this PR to follow that guidance.